flepimop/gempyor_pkg/src/gempyor/seeding.py

```diff
@@ -136,7 +134,7 @@ def get_from_config(self, sim_id: int, modinf) -> nb.typed.Dict:
             seeding = pd.DataFrame(columns=["date", "subpop"])
             return _DataFrame2NumbaDict(df=seeding, amounts=[], modinf=modinf)
         else:
-            raise NotImplementedError(f"unknown seeding method [got: {method}]")
+            raise NotImplementedError(f"Unknown seeding method [received: '{method}'].")
```

Comment:
```suggestion
            raise ValueError(f"Unknown seeding method given, '{method}'.")
```

Reply:
Was curious what your thoughts would be on leaving the `[received: '{method}']` format, because I've used this pretty widely across `gempyor` errors?

Reply:
I do not like the unnecessary punctuation since it makes it harder to read, can let someone else weigh in if you'd like another opinion though. However, this is also a two parter, the exception type needs to change here.

Reply:
Okay I understand. I can alter it. And yes already changed the exception type